### DISCUSSION TITLE
BioDrop Account Delete Request

### DISCUSSION THREAD

**Discussion by harisdev-netizen** [EddieHubCommunity/LinkFree#10168] [Q&A]
Hello, How do you delete your account in BioDrop? If it is not yet possible for users, admin kindly remove my account for now.

**Comment by eddiejaoude** [comment]
Delete is currently in a draft PR, so it will be available soon.

Please delete your json file, and we can deactivate your profile for now

**Comment by eddiejaoude** [comment]
Your profile has now been disabled

<img width="1668" alt="Screenshot 2024-01-22 at 20 49 49" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/4de33f29-4f93-4ae9-895b-564f40df8da5">
